supervisor/api/addons.py

```diff
@@ -213,7 +215,7 @@ async def info(self, request: web.Request) -> Dict[str, Any]:
             ATTR_MACHINE: addon.supported_machine,
             ATTR_HOMEASSISTANT: addon.homeassistant_version,
             ATTR_URL: addon.url,
-            ATTR_STATE: STATE_NONE,
+            ATTR_STATE: AddonState.UNKNOWN,
```

Comment:
The addon is initalized with `AddonState.UNKNOWN` we can use `addon.state` here

Reply:
It can be also an store add-on